corehq/apps/reports/standard/deployments.py

```diff
@@ -477,12 +501,51 @@ def _fmt_timestamp(timestamp):
 
         for row in table[1:]:
             # Last submission
-            row[len(location_colums) + 1] = _fmt_timestamp(row[len(location_colums) + 1])
-            # Last sync
             row[len(location_colums) + 2] = _fmt_timestamp(row[len(location_colums) + 2])
+            # Last sync
+            row[len(location_colums) + 3] = _fmt_timestamp(row[len(location_colums) + 3])
         result[0][1] = table
         return result
 
+    def get_location_column(self, user_doc, user_loc_dict):
```

Comment:
nit: this could be a private function.

Reply:
Addressed in [d6a20e8](https://github.com/dimagi/commcare-hq/pull/34744/commits/d6a20e8ee55103e1c5e211c6ae3034d5ecc12a42)